app/controllers/api/v1/proverbs_controller.rb

```diff
@@ -0,0 +1,73 @@
+module Api
+  module V1
+    class ProverbsController < ApplicationController
+      before_action :set_proverb, only: [:show, :update, :destroy, :edit]
+      before_action :authenticate
+
+      # GET /proverbs
+      # GET /proverbs.json
+      def index
+        @proverbs = Proverb.all
+
+        render json: @proverbs, status: :ok
+      end
+
+      # GET /proverbs/1
+      # GET /proverbs/1.json
+      def show
+        render json: @proverb, status: :ok
+      end
+
+      def edit
+        @proverb
+      end
+
+      def new
+        @proverb = Proverb.new
+      end
+
+      # POST /proverbs
+      # POST /proverbs.json
+      def create
+        data = proverb_params.merge!(user_id: @current_user.id)
+        @proverb = Proverb.create(data)
+        if @proverb
```

Comment:
This will always be true even if the resource was not created.
![image](https://cloud.githubusercontent.com/assets/11221027/18035138/121e745a-6d47-11e6-96a4-3c420f5ea75d.png)
You will want to check for errors on the `proverb` object.
Also you can use instance variable instead